### ISSUE TITLE
When updating your weights using the loss function, what dictates how much change the weights should have?

### ISSUE THREAD

**Issue by epassaro** [epassaro/aws-mls-c01#13]
### When updating your weights using the loss function, what dictates how much change the weights should have?

- [ ] Batch size
- [ ] Learning rate
- [ ] Initial weights
- [ ] Bias term

<details>
<summary><h3>Solución</h3></summary>

- [ ] Batch size
- [x] Learning rate
- [ ] Initial weights
- [ ] Bias term

</details>

<details>
<summary><h3>Razonamiento</h3></summary>
 
Una pregunta fácil tanto por la respuesta correcta como por las incorrectas.
</details>
